frontend/src/utilities/routeUtils.ts

```diff
@@ -0,0 +1,67 @@
+import { OdhRoute } from 'types';
+
+export const routes = {
```

Comment:
If this is the design we want -- we need to make sure all `<Route` references have a reference to this map -- otherwise it will be a maintenance annoyance